kybra/compiler/kybra_generate/src/main.rs

```diff
@@ -18,7 +18,7 @@ fn main() {
         Err(errors) => {
             eprintln!("Canister Compilation failed:");
             for error in errors {
-                eprintln!("{}", error)
+                eprintln!("{:#?}", error)
```

Comment:
Hmmm, I didn't have to do this in Azle. I'm wondering if you're missing a Debug trait, or doing Display differently or something. I think maybe we want to convert it into a string.

I don't really know so let's find a solution together. But I think one of us is doing something wrong.